src/infra/postgres/client/queries.go

```diff
@@ -0,0 +1,34 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
+)
+
+func QuerySearchIDs(ctx context.Context, client postgres.Client, idCol, whereCond string, whereArgs []interface{}, isDeleted bool, limit, offset uint64) ([]string, error) {
+	var ids = []string{}
+	var err error
+	var query string
+
+	switch {
+	case limit != 0 || offset != 0:
+		query = fmt.Sprintf("SELECT (array_agg(%s ORDER BY created_at ASC))[%d:%d] FROM secrets WHERE %s", idCol, offset+1, offset+limit, whereCond)
```

Comment:
How are your acceptance tests passing if the table is hardcoded to `secrets` here ? :thinking:

Reply:
becuase it was not passing xD. Good catch!